seirmo/models.py

```diff
@@ -1,7 +1,13 @@
+"""#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#"""
 #
 # ForwardModel Class
 #
 
+
```

Comment:
Could you also please add a `self` to the simulate method? Seems like I've missed quite a few slips when reviewing the other PR :D sorry!